src/v2/DevTools/setupTestWrapper.tsx

```diff
@@ -119,7 +119,7 @@ export const setupTestWrapperTL = <T extends OperationType>({
 }
 
 /**
- * @deprecated Use `setupTestWrapperTL`, which uses `@testing-library/react`
+ * @note See `setupTestWrapperTL`, which uses `@testing-library/react`
```

Comment:
If devs want to use RTL they can, but if they also want to use Enzyme that should be fine too. They're different things.